pkg/agent/clean/adunmigration/ldap.go

```diff
@@ -2,13 +2,16 @@ package adunmigration
 
 import (
 	"bytes"
+	"context"
 	"crypto/x509"
 	"fmt"
 	"os"
 	"regexp"
 	"strings"
 	"time"
 
+	"github.com/rancher/rancher/pkg/wrangler"
+
```

Comment:
No extra newline here.  I think our standard is one group for standard lib and one group for everything else.